### PR TITLE
[new release] odash (0.1.0)

### PR DESCRIPTION
**Pull request by roryqueue**
Odash: Lodash for Ocaml

- Project page: <a href="https://github.com/roryqueue/odash">https://github.com/roryqueue/odash</a>
- Documentation: <a href="https://roryqueue.github.io/odash/">https://roryqueue.github.io/odash/</a>

##### CHANGES:

* Initial release.
* Add identity, reduce, map, filter, reject, head, first, last, nth, tail, initial, flatten, concat, flatMap, forEach, each, forEachRight, eachRight, dropWhile, drop, dropRightWhile, dropRight, takeWhile, take, takeRightWhile, takeRight, slice, chunk, differenceBy, difference, fill, find, findLast, findIndex, findLastIndex, some, every, includesBy, includes, partition, sampleSize, sample, shuffle, size, simpleSortBy, sortBy, orderBy, join, uniqWith, uniqBy, uniq, intersectionWith, intersectionBy, intersection, xorWith, xorBy, xor, unionWith, unionBy, union, without, zip, and unzip.

### CODE REVIEW

File: packages/odash/odash.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"         {>= "4.02" & < "4.9"}
+  "dune"          {>= "1.11.4"}
+  "reason"        {>= "3.5.2"}
+]
```
Comment (kit-ty-kate) [inline packages/odash/odash.0.1.0/opam]:
```suggestion
  "ounit" {with-test}
]
```
According to this error message, I suspect this package requires ounit:
```
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
Processing  1/2: [odash: dune build]
+ /home/opam/.opam/4.02/bin/dune "build" "-p" "odash" "-j" "2" (CWD=/home/opam/.opam/4.02/.opam-switch/build/odash.0.1.0)
Processing  1/2: [odash: dune runtest]
+ /home/opam/.opam/4.02/bin/dune "runtest" "-p" "odash" "-j" "2" (CWD=/home/opam/.opam/4.02/.opam-switch/build/odash.0.1.0)
- File "test/dune", line 1, characters 33-38:
- 1 | (test (name test) (libraries Lib oUnit))
-                                      ^^^^^
- Error: Library "oUnit" not found.
- Hint: try: dune external-lib-deps --missing -p odash -j 2 @runtest
```

File: packages/odash/odash.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+bug-reports: "https://github.com/roryqueue/odash/issues"
+dev-repo: "git+https://github.com/roryqueue/odash.git"
+doc: "https://roryqueue.github.io/odash/"
+tags: [ "syntax" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"         {>= "4.02" & < "4.9"}
```
Comment (kit-ty-kate) [inline packages/odash/odash.0.1.0/opam]:
```suggestion
  "ocaml"         {>= "4.02" & < "4.09"}
```

**Comment by roryqueue** [comment]
thanks for your help @kit-ty-kate

**Comment by roryqueue** [comment]
oh that failure's my bad, I replaced the release separately. will close and reopen

**Pull request closed without merging**